examples/pytorch/vision/Face_Detection/train.py

```diff
@@ -29,9 +29,6 @@ def str2bool(v):
                     default='face',
                     choices=['hand', 'face', 'head'],
                     help='Train target')
-parser.add_argument('--basenet',
```

Comment:
why are you removing this option?

Reply:
we are not using a pretrained model like s3fd does